app/controllers/admin/reservations/reservation_loans_controller.rb

```diff
@@ -75,13 +81,14 @@ def destroy
       private
 
       def render_form_with_error(message)
+        @sound_type = failure_sound_path
         @reservation_loan = ReservationLoan.new
         @reservation_loan.errors.add(:reservable_item_number, message)
         render_form
       end
 
       def render_form
-        render partial: "admin/reservations/reservation_loans/form", locals: {reservation: @reservation, reservation_loan: @reservation_loan}, status: :unprocessable_entity
+        render_turbo_response :create_error, status: :unprocessable_entity
```

Comment:
This `create_error` pattern seemed to be hanging around other parts of the app so I just copied that.